src/shared_gui_components/BuildingComponentDialogCentralWidget.cpp

```diff
@@ -210,11 +238,11 @@ void BuildingComponentDialogCentralWidget::setTid(const std::string& filterType,
   m_collapsibleComponentList->setText(title);
 
   // the total number of results
-  int lastTotalResults = remoteBCL.lastTotalResults();
+  int lastTotalResults = m_allResponses.size();
   m_collapsibleComponentList->setNumResults(lastTotalResults);
```

Comment:
What do `numResults` and `numPages` do?

Reply:
I have refractored the code to remove these variables